lib/chef/telemetry.rb

```diff
@@ -0,0 +1,76 @@
+#
+# Author:: Ranjib Dey (<ranjib@linux.com>)
+# Copyright:: Copyright (c) 2015 Chef Inc.
```

Comment:
While you are free to assign copyright to Chef Software Inc. (though you should fix the name), you are under no obligation to do so as this was not produced under work-for-hire that I'm aware of. If this was produced at $DAYJOB you should check if they retain copyright, otherwise by default it stays with you personally.